mpu6050_userspace/mpu6050_get_temp_C_and_F.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+function view_help {
+	echo "-h, --help	help information"
+	echo "This programm for print temperature from mpu6050 in Celsius and Fahrenheit"
+}
+
+function conv_16_bit_hex_dec_signed {
+    res=$(printf "%d" "0x$1")
+    (( res > 32767 )) && (( res -= 65536 ))
+    echo $res
+}
+function get_temperature {
+    H_BYTE_MPU6060=$(i2cget -y 0 0x68 0x41 b)
+    L_BYTE_MPU6060=$(i2cget -y 0 0x68 0x42 b)
+    TWO_BYTE_HEX="${H_BYTE_MPU6060:2}${L_BYTE_MPU6060:2}"
+    TWO_BYTE_DEC_convert=$(conv_16_bit_hex_dec_signed $TWO_BYTE_HEX )
+#    echo "TWO_BYTE_DEC=$TWO_BYTE_DEC_convert"
+    echo "Temp TWO_BYTE_HEX $( echo "scale=3; $TWO_BYTE_DEC_convert/340+36.53" | bc)"
+    echo "Temp Fahrenheit $(echo "scale=3; (($TWO_BYTE_DEC_convert/340+36.53)*9)/5+32" | bc)"
+
+}
```

Comment:
Can You implement the conversation without additional call of printf and using out from i2cget with option -w

Reply:
The changes are not implemented.